src/shared/lib/CancerHotspotsUtils.ts

```diff
@@ -19,7 +18,7 @@ export async function fetchHotspotsData(mutationData: MobxPromise<Mutation[]>,
         return [];
     }
 
-    const genomicLocations: GenomicLocation[] = _.uniq(_.map(mutationDataResult, extractGenomicLocation));
+    const genomicLocations: GenomicLocation[] = uniqueGenomicLocations(mutationDataResult);
```

Comment:
apparently `_.uniq` does not work the way I thought it would

Reply:
do tell?

Reply:
@onursumer curious as well

Reply:
Okay, according to lodash documentation it uses this function to compare objects: http://ecma-international.org/ecma-262/7.0/#sec-samevaluenonnumber

Seems like it doesn't perform deep equality comparison, that's why it wasn't removing duplicates.

Reply:
ah, i see